api/src/test/java/org/openmrs/api/PersonServiceTest.java

```diff
@@ -1730,6 +1753,23 @@ public void voidRelationship_shouldVoidRelationshipAndSetVoidedByToGivenUserIfGi
 		assertEquals(voidedRelationship.getVoidedBy(), user);
 	}
 	
+	@Test
+	public void voidRelationship_shouldVoidRelationshipAndSetVoidedByToGivenUserIfGivenRelationshipHasNullVoidedBy() {
```

Comment:
this test already exists. please remove it

Reply:
@teleivo Ok I will remove the test. Thanks